src/status_im/ios/core.cljs

```diff
@@ -38,7 +39,7 @@
                       #(when-not (= 0 @keyboard-height)
                          (dispatch [:set :keyboard-height 0])))
         (.hide react/splash-screen)
-        (.useScreens rn-dependencies/react-native-screens)
+        (ocall rn-dependencies/react-native-screens "enableScreens")
```

Comment:
are you sure we need this with rn5 ?

Reply:
We can remove it, an artifact from rn4 :+1: